packages/lodestar/src/chain/chain.ts

```diff
@@ -110,7 +110,7 @@ export class BeaconChain implements IBeaconChain {
     this.logger = logger;
     this.metrics = metrics;
     this.genesisTime = anchorState.genesisTime;
-    this.anchorSlot = anchorState.slot;
+    this.anchorSlot = anchorState.latestBlockHeader.slot;
```

Comment:
`anchorSlot` is too vague and can be confusing since this is on the public facing API of the BeaconChain object. Since this is used just by an internal module use a long but precise name like:
`anchorStateLatestBlockSlot`